study.md

```diff
@@ -66,7 +67,8 @@ Modify `dale` so that the value of the key `family_name` is "Gribble" and the
 value of the key `given-name` is "Dale".
 
 ```ruby
-# your answer here
+dale[:family_name]="gribble".capitalize!
+dale[:"given-name"]=:"dale".capitalize!
```

Comment:
This will cause an error since you're attempting to invoke `capitalize!` on the symbol `:dale`.